ts/src/mexc.ts

```diff
@@ -1091,87 +1091,48 @@ export default class mexc extends Exchange {
             const id = this.safeString (currency, 'coin');
             const code = this.safeCurrencyCode (id);
             const name = this.safeString (currency, 'name');
-            let currencyActive = false;
```

Comment:
@ttodua btw how is this related to createOrders? §

Reply:
tests didnt pass for fetchCurrencies and I had to edit them too, but i can move it